app/src/screens/teams/TeamManageScreen.tsx

```diff
@@ -17,9 +22,22 @@ type TeamManageScreenProps = object &
 export const TeamManageScreen: React.FC<TeamManageScreenProps> = ({
   navigation,
 }) => {
+  const userInfo = useUserProfileQuery();
+  const membersList = useShowMembersOfTeamQuery(userInfo.data?.current_team_id);
+
   const [switched, setSwitched] = useState<boolean>(false);
```

Comment:
Jeżeli przekazujesz początkową wartość do `useState` to TypeScript na jej podstawie potrafi sam określić typ przechowywanego obiektu, więc `<boolean>` w tym miejscu można pominąć